plugins/modules/cloud/misc/proxmox_snap.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_snap
+short_description: snapshot management of instances in Proxmox VE cluster
+description:
+  - allows you to create/delete snapshots from instances in Proxmox VE cluster
+options:
+  api_host:
+    description:
+      - the host of the Proxmox VE cluster
```

Comment:
```suggestion
      - The host of the Proxmox VE cluster.
```
Same for the other descriptions. Every paragraph (list item) should be a (or multiple) correct sentence(s).

Reply:
fixed!